CHANGES/502.feature

```diff
@@ -0,0 +1,2 @@
+Added support for pushing image signatures to the Pulp Registry. The signatures can be pushed by
```

Comment:
can you extend existing docs? for example add another section to the signing workflow and mention another way how get signature into pulp registry.